doc/adr/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,24 @@
+<!-- This is the template in Documenting architecture decisions - Michael Nygard link: https://cognitect.com/blog/2011/11/15/documenting-architecture-decisions -->
```

Comment:
Is this an ADR or is it a template? If it's a template, shouldn't it say so in the file name? To me the file name suggests this is an actual ADR. If this is an actual ADR for "Record architectural decisions", the content should be changed.

Reply:
From the original `adr-tools` documentation it says `adr-init` creates an /adr folder "containing the first ADR, which records that you are using ADRs to record architectural decisions", which the `adr-tools-python` is replicating.

I was assumed it was both ADR and "template" since the ADR was almost the same as the Nygard template I linked, but I believe I am mistaken as its not a template, only an ADR. I will change my comment to remove template and move that information to the explanation.md section.